pyproject.toml

```diff
@@ -34,6 +34,7 @@ default-groups = ["dev"]
 
 [dependency-groups]
 dev = [
+  "fastapi[standard]==0.123.5",
   "ruff==0.14.7",
```

Comment:
I added this as a dev dependency as the app seems to work fine without it when deployed in docker - it uses `exec uvicorn \
    --host 0.0.0.0 \
    --port 5000 \
    --forwarded-allow-ips='*' \
    asgi:app \
    --workers 1 \`

The local command `uv run fastapi dev asgi.py --port 5000` did not work until I did this.

It might be fine to add as a dependency - as far as I can tell adding `[standard]` brings in the cli